src/screens/history/components/HistorySkeletonLoading.tsx

```diff
@@ -16,7 +16,7 @@ const HistorySkeletonLoading: React.FC<Props> = ({ theme }) => {
 
   const renderLoadingChapter = (item: number, index: number) => {
     return (
-      <>
```

Comment:
Index shouldn't be used as a key. You can read more [here](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/).